frontend/src/pages/Interventions/FirstActivity.tsx

```diff
@@ -100,6 +100,7 @@ const FirstActivity = ({
                 top={20}
                 delay={2000}
                 onClick={updateIntervention}
+                canBeShown
```

Comment:
ig technically we don't really need canBeShown for first activity and fourth activity but I think it's okay to just leave it